README.md

```diff
@@ -2,10 +2,7 @@
 
 **Shhgit finds secrets and sensitive files across GitHub code and Gists committed in *near* real time by listening to the [GitHub Events API](https://developer.github.com/v3/activity/events/).**
 
-<p align="center">
-<img src="https://www.darkport.co.uk/assets/img/shhgit.png" alt="shhgit" width="200" />
-</p>
-
+![shhgit](images/shhgit.png)
```

Comment:
This is not centered anymore but at least present. Let me know what you think @eth0izzle